src/test/java/com/kalessil/phpStorm/phpInspectionsEA/languageConstructions/ProtectedWithFinalClassInspectorTest.java

```diff
@@ -0,0 +1,20 @@
+package com.kalessil.phpStorm.phpInspectionsEA.languageConstructions;
+
+import com.intellij.codeInsight.intention.IntentionAction;
+import com.intellij.testFramework.fixtures.CodeInsightFixtureTestCase;
+import com.kalessil.phpStorm.phpInspectionsEA.inspectors.languageConstructions.ProtectedWithFinalClassInspector;
+
+public class ProtectedWithFinalClassInspectorTest extends CodeInsightFixtureTestCase {
+    public void testIfFindsAllPatterns() {
+        myFixture.configureByFile("fixtures/codeSmell/protected-with-final-class.php");
+        myFixture.enableInspections(ProtectedWithFinalClassInspector.class);
```

Comment:
I started cleaning up tests, here should be new ProtectedWithFinalClassInspector() to not introduce warnings.